app/controllers/WebchatController.scala

```diff
@@ -91,7 +94,11 @@ class WebchatController @Inject()(appConfig: AppConfig,
   }
 
   def onlineServicesHelpdesk: Action[AnyContent] = Action.async { implicit request =>
-    Future.successful(Ok(onlineServiceHelpdeskView()))
+    if (config.showOSHCUI) {
+      Future.successful(Ok(onlineServicesHelpdeskCUIView()))
```

Comment:
@sanjayshukla2205 instead of presenting the CUI view here, could we redirect to the OSH CUI controller instead? This way we can see the redirects in Kibana. Thanks

Reply:
Hi Lorenzo  thanks for the review and i have added the test as per your recommendations